tests/v2/validation/rbac/rbac_psa_test.go

```diff
@@ -250,8 +253,34 @@ func (rb *PSATestSuite) ValidateAdditionalPSA(role string) {
 	}
 }
 
+func (rb *PSATestSuite) ValidateEditPsactCluster(role string) {
+	clusterType, err := editPsactCluster(rb.nonAdminUserClient, rb.clusterName, defaultNamespace, "")
+	switch role {
+	case roleOwner, restrictedAdmin:
+		require.NoError(rb.T(), err)
+		_, err = psadeploy.CreateNginxDeployment(rb.nonAdminUserClient, rb.clusterID, "")
+		require.NoError(rb.T(), err)
+
+		_, err = editPsactCluster(rb.nonAdminUserClient, rb.clusterName, defaultNamespace, "rancher-restricted")
```

Comment:
lets add this for cluster members also. So we are verifying the test for all the users.